### PR TITLE
Issue #7163: aligned javadoc/xdoc for RegexpSingleline

### PR DESCRIPTION
**Pull request by pbludov**
Issue #7163 (part of #5750)

No changes in the code.

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java [hunk 1]
```diff
@@ -65,40 +138,45 @@ protected void processFiltered(File file, FileText fileText) {
...
+     * @param maximum the maximum number of matches required in each file.
      */
     public void setMaximum(int maximum) {
         this.maximum = maximum;
     }
 
     /**
-     * Set whether to ignore case when matching.
-     * @param ignoreCase whether to ignore case when matching.
+     *Setter to control whether to ignore case when searching.
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java]: Missing a starting space.

Comment (rnveach) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java]: I am not seeing that in the discussion there. Whole issue is about where asterisk to aligned.
The issue here is the space after the asterisk, not the asterisk itself.

Comment (pbludov) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java]: You are right, there should be another `JavadocMissingWhitespaceCheck`.

Comment (pbludov) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java]: > Missing a starting space.

done

Comment (romani) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java]: JavadocMissingWhitespaceAfter AsteriskCheck

Comment (rnveach) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java]: @pbludov I am not seeing a new issue created for this.

Comment (pbludov) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java]: > @pbludov I am not seeing a new issue created for this.

Done: https://github.com/checkstyle/checkstyle/issues/7183

Comment (romani) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java]: Let's not block this PR by this off topic discussion

Comment (rnveach) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java]: @pbludov The code in this PR has still not been fixed.

Comment (pbludov) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java]: done

**Comment by rnveach** [comment]
Waiting on CI to pass.

**Comment by pbludov** [comment]
> Waiting on CI to pass.

It fails with `Execution timeout`

**Pull request merged**